### ISSUE TITLE
Image pixels not aligned to their expected coordinates

### ISSUE THREAD

**Issue by pijyoi** [pyqtgraph/pyqtgraph#431]
When you zoom in to look at the pixels, their boundaries are not at the expected integer coordinates. The misalignment is most pronounced at the bottom-right pixels.

My actual use-case is to assign frequency coords to a spectrogram, but the default integer coords were already misaligned.

```python
import numpy as np
import pyqtgraph as pg

app = pg.mkQApp()

x = np.random.random((5,5))

pw = pg.plot()
plotitem = pw.getPlotItem()

img = pg.ImageItem(x)
plotitem.addItem(img)

app.exec_()
```

<img width="960" alt="capture" src="https://cloud.githubusercontent.com/assets/2657027/22141118/530adaf2-df2b-11e6-8ac7-70fa651df0af.PNG">

**Comment by pijyoi** [comment]
The misalignment is tested to be present in:
PyQt5 5.9.2 (Windows x64)

The misalignment is *not* present in:
PyQt5 5.13.0 (Windows x64)
PySide2 5.12.1 (Windows x64)

**Comment by j9ac9k** [comment]
@pijyoi I know this issue is from forever ago; but as we now only support Qt 5.12+ do you think we should still keep it open and try and make a fix?

**Comment by pijyoi** [comment]
I don't think we should try to make any fix / workaround.

1. It's fixed upstream somewhere between 5.9 LTS and 5.12 LTS
2. Any downstream workaround, if possible at all, would require knowledge of the exact version when it got fixed upstream.

**Comment by j9ac9k** [comment]
Given 5.12 is the minimum version we support, I'll close this issue.  👍
